simulation-model/simulation-model_shacl.ttl

```diff
@@ -0,0 +1,317 @@
+            sh:datatype xsd:dateTime ;
+            sh:description "Model Release Date in ISO8601"@en ;
+            sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:message "Validation of model.releasedate failed!"@en ;
+            sh:name "model.releasedate"@en ;
+            sh:order 5 ;
+            sh:path simulation-model:model.releasedate ],
+        [ skos:example "In Development" ;
+            skos:example "Released" ;
+            skos:example "Decommissioned" ;
+            sh:datatype xsd:string ;
+            sh:description "The status of the model version released in the life cycle of the model."@en ;
+            sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:message "Validation of model.releasestatus failed!"@en ;
+            sh:name "model.releasestatus"@en ;
+            sh:order 6 ;
+            sh:path simulation-model:model.releasestatus ],
+        [ skos:example "No use restrictions" ;
+            sh:datatype xsd:string ;
+            sh:description "Legal or contractual restrictions on the use and distribution of the model that have to be respected. Applicable Licenses should be mentioned here."@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of model.use-restrictions failed!"@en ;
+            sh:name "model.use-restrictions"@en ;
+            sh:order 7 ;
+            sh:path simulation-model:model.use-restrictions ],
+        [ skos:example "7ae4-210f-dead-beef-042" ;
+            sh:datatype xsd:string ;
+            sh:description "For FMU-based models, this should be the GUID of the FMU."@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of model.guid failed!"@en ;
+            sh:name "model.guid"@en ;
+            sh:order 8 ;
+            sh:path simulation-model:model.guid ],
+        [ skos:example "HAD function validation in OEM environment at object-list level" ;
+            sh:datatype xsd:string ;
+            sh:description "Purpose for which the model has been built/validated."@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of model.purpose failed!"@en ;
+            sh:name "model.purpose"@en ;
+            sh:order 9 ;
+            sh:path simulation-model:model.purpose ],
+        [ skos:example "model is purely object-list driven, weather effects are not modelled." ;
+            sh:datatype xsd:string ;
+            sh:description "Description of the modelling approach taken."@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of model.modelling-approach failed!"@en ;
+            sh:name "model.modelling-approach"@en ;
+            sh:order 10 ;
+            sh:path simulation-model:model.modelling-approach ] ;
+    sh:targetClass simulation-model:Content .
+
+simulation-model:QuantityShape a sh:NodeShape ;
+    sh:closed false ;
+    sh:ignoredProperties ( rdf:type ) ;
+    sh:property [ skos:example "800" ;
+            sh:datatype xsd:positiveInteger ;
```

Comment:
Good point. I just read through the list of possible `xsd` attributes and found that `xsd:positiveInteger` is probably most readable but since `unsignedInt` is used elsewhere and it has the same meaning I've changed it.